src/components/BudgetOverviewCard/BudgetOverviewCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { AccountBalance } from '@material-ui/icons';
+import { sumBy } from 'lodash';
+import * as React from 'react';
+import { FC } from 'react';
+import { FieldOverviewCard } from '../FieldOverviewCard';
+import { useCurrencyFormatter } from '../Formatters/useCurrencyFormatter';
+import { BudgetItemFragment } from './BudgetItem.generated';
+
+export interface BudgetItemCardProps {
+  budget?: BudgetItemFragment;
+  className?: string;
+}
+
+export const BudgetOverviewCard: FC<BudgetItemCardProps> = ({
+  className,
+  budget,
+}) => {
+  const currencyFormatter = useCurrencyFormatter();
+  const amount = sumBy(budget?.records, (record) => record.amount.value ?? 0);
+
+  return (
+    <FieldOverviewCard
+      className={className}
+      title="Project Budget"
+      viewLabel="Budget history"
```

Comment:
I know designs said this, but we won't be showing history on this page for initial release.
```suggestion
      viewLabel="Budget Details"
```